README.md

```diff
@@ -473,6 +473,7 @@ Table of Contents
   * [bitrise.io](https://www.bitrise.io/) — A CI/CD for mobile apps, native or hybrid. With 200 free builds/month 10 min build time and two team members. OSS projects get 45 min build time, +1 concurrency and unlimited team size.
   * [buddy.works](https://buddy.works/) — A CI/CD with 5 free projects and 1 concurrent runs (120 executions/month)
   * [buddybuild.com](https://www.buddybuild.com/) — Build, deploy and gather feedback for your iOS and Android apps in one seamless, iterative system
+  * [Buildkite](https://buildkite.com/) — Free for open source projects, students, and teaching organizations.  The CI pipelines run on your own infrastructure.
```

Comment:
it's essentially self hosted software, dont think it applies to this list.

We have others that manage your own infra - but the feature set they provide on their infra is significant, this is minimal, would rather skip it there are many more useful free CI tools than this.